app/services/post_status_service.rb

```diff
@@ -67,7 +70,7 @@ def process_status!
     end
 
     process_hashtags_service.call(@status)
-    process_mentions_service.call(@status)
+    process_mentions_service.call(@status, @circle)
```

Comment:
This is slightly awkward, at first I thought this didn't make sense, but given that we're implementing that through “silent mentions”… i guess this does make sense.